src/patient-search/patient-search-result/patient-search-result.component.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import styles from "./patient-search-result.component.css";
+import dayjs from "dayjs";
+
+export default function PatientSearchResults(props: PatientSearchResultsProps) {
+  return props.patients.map((patient, index) =>
+    renderPatient(patient, index + 1)
+  );
+
+  function renderPatient(patient, resultNumber) {
+    const preferredIdentifier =
+      patient.identifiers.find(i => i.preferred) || patient.identifiers[0];
+
+    return (
+      <div
+        key={patient.display}
+        className={styles.patientResult}
+        data-testid="searchResult"
+      >
+        <span className={styles.resultNumber}>{resultNumber}</span>
+        <div className={styles.patientCard}>
+          <div className={styles.patientNameContainer}>
+            <span className={styles.patientName}>{patient.person.display}</span>
+            <button className="omrs-unstyled">
+              <svg className="omrs-icon" fill="var(--omrs-color-interaction)">
```

Comment:
Looking at this reminded me that we should add `<title>` elements to the svgs in the styleguide, for accessibility purposes.

Reply:
Created a ticket for this [here](https://issues.openmrs.org/browse/MF-55)